packages/ember-runtime/lib/computed/computed_macros.js

```diff
@@ -20,7 +20,10 @@ function expandPropertiesToArray(properties) {
   }
 
   for (let i = 0; i < properties.length; i++) {
-    expandProperties(properties[i], extractProperty);
+    let property = properties[i];
+    assert('Dependent keys passed to Ember.computed.and() or Ember.computed.or() can\'t have spaces.', property.indexOf(' ') < 0);
```

Comment:
Looks good, but I'd much prefer the message to be tailored to the specific type of computer macro that was being used. Is this same message fired for ither macros? If it is, can we somehow pass in the macro being used and use that in the message directly?